pkg/smart/parseDisk.go

```diff
@@ -0,0 +1,40 @@
+package smart
+
+import (
+	"bytes"
+	"regexp"
+)
+
+var diskScanRegexp *regexp.Regexp
+
+func init() {
+	diskScanRegexp = regexp.MustCompile(`^(\/dev\/(\w|\d)+)(\s?(\w|\W)+)?.?$`)
+}
+
+func parseDisks(buf *bytes.Buffer) ([]string, error) {
+	var disks []string
+
+	for {
+		s, err := buf.ReadString('\n')
+		if s != "" {
+			disks = append(disks, s)
+		}
+
+		if err != nil {
```

Comment:
Probably this check should also be altered: 
err can have  EOF value here, which will end the loop. But also it can have some error and we probably should handle it..